### PR TITLE
Feature/devsu 1637 upgrade mui

### PR DESCRIPTION
**Pull request by Anna-log7**
I've upgraded MUI to v5 in this PR by running the codemod on their site: https://mui.com/guides/migration-v4/
The build will fail because node 10 is still a task (that we need to remove). This PR won't be included in the 6.6.0 release

### CODE REVIEW

File: app/components/DataTable/components/ActionCellRenderer/index.jsx [hunk 1]
```diff
@@ -78,131 +78,129 @@ function ActionCellRenderer(params) {
     setAnchorEl(null);
   };
 
-  return (
-    <>
-      {canViewDetails && (
+  return <>
```
Comment (kttkjl) [inline app/components/DataTable/components/ActionCellRenderer/index.jsx:81]: think auto fix messed the styling on this

File: app/components/DataTable/components/GeneCellRenderer/index.jsx [hunk 2]
```diff
@@ -16,45 +16,43 @@ function GeneCellRenderer(params) {
...
   const [showGeneViewer, setShowGeneViewer] = useState(false);
 
-  return (
-    <>
-      {value && value.split(/\s*::\s*|,\s/).map((val, index) => (
-        <React.Fragment key={index}>
-          {index > 0 && (
-            <span>
-              {value.includes(' :: ') ? ' :: ' : ', '}
+  return <>
```
Comment (kttkjl) [inline app/components/DataTable/components/GeneCellRenderer/index.jsx:19]: here as well, auto fix messed with styling

File: app/views/ReportView/components/MutationBurden/components/TabCards/index.tsx [hunk 3]
```diff
@@ -169,72 +169,70 @@ const TabCards = ({
...
-                    )}
-                  </Card>
-                )}
-              </React.Fragment>
-            );
-          })}
-      </div>
-    </>
-  );
+  return <>
```
Comment (kttkjl) [inline app/views/ReportView/components/MutationBurden/components/TabCards/index.tsx:172]: auto styling problem

File: app/components/Sidebar/index.tsx [hunk 4]
```diff
@@ -210,7 +210,7 @@ const Sidebar = (): JSX.Element => {
           {drawer}
         </Drawer>
       </Hidden>
-      <Hidden xsDown>
+      <Hidden smDown>
```
Comment (kttkjl) [inline app/components/Sidebar/index.tsx:213]: it doesn't look like xs v4 is diff from xs v5? why was this change here

**Review by kttkjl — CHANGES_REQUESTED** [review]

File: app/components/Sidebar/index.tsx (hunk 4 above)
Comment (Anna-log7) [reply app/components/Sidebar/index.tsx:213]: Not sure, I'll look into it

Comment (Anna-log7) [reply app/components/Sidebar/index.tsx:213]: https://mui.com/guides/migration-v4/#theme there was changes, but I don't know if they necessarily apply here. `xsDown` seems to mean less than 0px, so I think this change is correct. I tested it and `smDown` works for the behaviour I want.

File: app/views/ReportView/components/MutationBurden/components/TabCards/index.tsx (hunk 3 above)
Comment (Anna-log7) [reply app/views/ReportView/components/MutationBurden/components/TabCards/index.tsx]: fixed all of these

**Review by elewis2 — APPROVED** [review]

**Review by kttkjl — APPROVED** [review]
